src/runtime_src/core/pcie/driver/linux/xocl/subdev/ert_user.c

```diff
@@ -810,7 +810,7 @@ ert_post_process(struct xocl_ert_user *ert_user, struct xrt_ert_command *ecmd)
 {
 	struct ert_access_valid_cmd *cmd = NULL;
 	uint32_t offset = 0;
-	if (likely(!ert_special_cmd(ecmd)) && cmd_opcode(ecmd) != ERT_ACCESS_TEST)
+	if (likely(!ert_special_cmd(ecmd)) && (cmd_opcode(ecmd) != ERT_ACCESS_TEST || cmd_opcode(ecmd) != ERT_SK_START))
```

Comment:
I am wondering if it makes more sense to fix it in ert_special_cmd()

Reply:
Besides that, this line is too long. :)

Reply:
Then probably need to add both ERT_ACCESS_TEST and ERT_SK_START to ert_special_cmd()?

Reply:
I don't know the background of ERT_ACCESS_TEST and why it is handled differently.  But I think ERT_CU_START and ERT_SK_START belong to same category.